src/Features/ERDDAP/Platform/Observations/Condition/index.tsx

```diff
@@ -1,64 +1,80 @@
+/**
+ * Display all time series for a specific standard name
+ */
 import * as React from "react"
 import { Col, Row } from "reactstrap"
 
 import { LargeTimeSeriesChart } from "components/Charts"
 import { naturalBounds } from "Shared/dataTypes"
-import { UnitSystem } from "Features/Units/types"
+import { useUnitSystem } from "Features/Units"
 
-import { PlatformFeatureWithDatasets, PlatformDataset } from "../../../types"
-import { ErddapDatasetFinder, ErddapDatasetLoader } from "../../Dataset"
+import { UseDataset } from "../../../hooks"
+import { PlatformFeature, PlatformTimeSeries } from "../../../types"
+import { DataTimeSeries } from "Shared/timeSeries"
+import { UnitSystem } from "Features/Units/types"
 
 interface Props {
-  platform: PlatformFeatureWithDatasets
+  /** Platform to display */
+  platform: PlatformFeature
+  /** Standard name to display */
   standardName: string
-  unit_system: UnitSystem
 }
 
-export const ErddapObservedCondition: React.SFC<Props> = props => (
-  <ErddapDatasetFinder platform={props.platform} standardName={props.standardName}>
-    {({ datasets }) => (
-      <ErddapDatasetLoader platformId={props.platform.id as string} datasets={datasets}>
-        <ObservedConditionsDatasets {...props} datasets={datasets} />
-      </ErddapDatasetLoader>
-    )}
-  </ErddapDatasetFinder>
-)
+/**
+ *
+ * @param platform
+ * @param standardName
+ */
+export const ErddapObservedCondition: React.FunctionComponent<Props> = ({ platform, standardName }) => {
+  const unit_system = useUnitSystem()
```

Comment:
Codacy found an issue: [Identifier 'unit_system' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)